From 66029529fa0f0e2d16999f22294822deeec5f60b Mon Sep 17 00:00:00 2001
From: Alejandro Visiedo <avisiedo@redhat.com>
Date: Thu, 11 Jun 2020 00:36:04 +0200
Subject: [PATCH] systemtap: Missing a comma

sssd_functions.stp was missing a comma.

Thanks to William Cohen for reporting the issue and the patch to fix it.

https://bugzilla.redhat.com/show_bug.cgi?id=1840194

Resolves: https://github.com/SSSD/sssd/issues/5201

Reviewed-by: Pawel Polawski <ppolawsk@redhat.com>
---
 src/systemtap/sssd_functions.stp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/systemtap/sssd_functions.stp b/src/systemtap/sssd_functions.stp
index 1eb140ccf..01f553177 100644
--- a/src/systemtap/sssd_functions.stp
+++ b/src/systemtap/sssd_functions.stp
@@ -7,7 +7,7 @@ global TARGET_ID=0, TARGET_AUTH=1, TARGET_ACCESS=2, TARGET_CHPASS=3,
 global METHOD_CHECK_ONLINE=0, METHOD_ACCOUNT_HANDLER=1, METHOD_AUTH_HANDLER=2,
        METHOD_ACCESS_HANDLER=3, METHOD_SELINUX_HANDLER=4, METHOD_SUDO_HANDLER=5,
        METHOD_AUTOFS_HANDLER=6, METHOD_HOSTID_HANDLER=7, METHOD_DOMAINS_HANDLER=8,
-       METHOD_RESOLVER_HANDLER=9 METHOD_SENTINEL=10
+       METHOD_RESOLVER_HANDLER=9, METHOD_SENTINEL=10
 
 function acct_req_desc(entry_type)
 {
-- 
2.21.3

